internal/marshal/movie_omdb.go

```diff
@@ -0,0 +1,31 @@
+package marshal
+
+import (
+	"strconv"
+
+	"github.com/maestre3d/academy-go-q12021/internal/aggregate"
+	"github.com/maestre3d/academy-go-q12021/internal/valueobject"
+)
+
+const movieDirectorDelimeterPatternOmdb = ", "
+
+// MovieOmdb OMDb API Movie schema
+type MovieOmdb struct {
+	Title    string `json:"Title"`
+	Year     string `json:"Year"`
+	Director string `json:"Director"`
+	IMDbID   string `json:"imdbID"`
+}
+
+// UnmarshalMovieOmdb parses the given MovieOMDb into an aggregate Movie
+func UnmarshalMovieOmdb(movieOmdb MovieOmdb, movie *aggregate.Movie) (err error) {
+	movie.DisplayName = valueobject.DisplayName(movieOmdb.Title)
+	movie.Directors = unmarshalMovieDirectors(movieDirectorDelimeterPatternOmdb, movieOmdb.Director)
+	year, _ := strconv.Atoi(movieOmdb.Year)
```

Comment:
Handle all the possible errors.

Reply:
Will handle the error, yet I wanted to mention this wont cause a panic. Thus, we might be able to handle non-integer years by using the built-in value (0) by the given ignore example.